utils/create_merge_ids.py

```diff
@@ -18,42 +19,24 @@ def create_cset_article_id(idx: int):
     return f"carticle_{zero_padding}{idx}"
 
 
-def get_connected_edges(adj_list: dict, key: str) -> set:
+def get_connected_articles(adj_list: dict, key: str) -> set:
     """
     Given a dict where a key-value pair corresponds to an article match and a particular article `key`,
     returns a set of articles matched to `key`.
     :param adj_list: a dict of key-value pairs corresponding to matched articles
     :param key: an article to match in `adj_list`
     :return: a set of matched articles
     """
-    conn_edges = {key}
+    conn_articles = {key}
     to_explore = adj_list[key]
     while len(to_explore) > 0:
         v = to_explore.pop()
-        if v not in conn_edges:
-            conn_edges.add(v)
+        if v not in conn_articles:
+            conn_articles.add(v)
             to_explore = to_explore.union(
-                {k for k in adj_list[v] if k not in conn_edges}
+                {k for k in adj_list[v] if k not in conn_articles}
             )
-    return conn_edges
-
-
-def get_usable_ids(ids_dir: str) -> set:
```

Comment:
As above - this code was needed because of the possibility of introducing obsolete orig ids from the simhash index. Because we're getting rid of simhash, this is unnecessary.